spec/features/applicant_page_spec.rb

```diff
@@ -1,260 +0,0 @@
-require "rails_helper"
-
-RSpec.describe "Applicant Page" do
-  let(:applicant_header) { I18n.t("estimate_flow.applicant.heading") }
-
-  describe "errors" do
-    before do
```

Comment:
Personally I'm feeling very nervous that we have deleted over 1,000 lines of test just in this block. I'm totally not confident that some important test hasn't been deleted as part of this (and presumably others in this PR)